solr/core/src/test/org/apache/solr/TestTolerantSearch.java

```diff
@@ -104,10 +104,7 @@ public static void destroyThings() throws Exception {
       collection2.close();
       collection2 = null;
     }
-    if (null != jetty) {
-      jetty.stop();
-      jetty = null;
-    }
+    solrClientTestRule.reset();
```

Comment:
i mentally thought that by using a test rule, some of this clean up stuff would just be handled by us, because it's a rule?